sci-physics/rivet/rivet-3.1.5.ebuild

```diff
@@ -62,8 +62,8 @@ src_configure() {
 	PREFIX_YODA=$(yoda-config --prefix) || die
 	PREFIX_FJ=$(fastjet-config --prefix) || die
 	econf \
-		$(usex hepmc2 "--with-hepmc=/usr" "") \
-		$(usex hepmc3 "--with-hepmc3=/usr" "") \
+		$(usex hepmc2 "--with-hepmc=${SYSROOT}/usr" "") \
+		$(usex hepmc3 "--with-hepmc3=${SYSROOT}/usr" "") \
 		--with-yoda=$PREFIX_YODA \
```

Comment:
We use ${PREFIX_YODA} style in Gentoo: https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101